packages/react-native-gesture-handler/src/web/tools/InteractionManager.ts

```diff
@@ -31,20 +35,22 @@ export default class InteractionManager {
 
     if (config.simultaneousHandlers) {
       const simultaneousHandlers: number[] = [];
-      config.simultaneousHandlers.forEach((otherHandler: Handler): void => {
-        if (typeof otherHandler === 'number') {
-          simultaneousHandlers.push(otherHandler);
-        } else {
-          simultaneousHandlers.push(otherHandler.handlerTag);
+      config.simultaneousHandlers.forEach(
+        (otherHandler: Handler | number): void => {
+          if (typeof otherHandler === 'number') {
+            simultaneousHandlers.push(otherHandler);
+          } else {
+            simultaneousHandlers.push(otherHandler.handlerTag);
+          }
```

Comment:
```suggestion
(otherHandler: Handler | number): void => {
  simultaneousHandlers.push(typeof otherHandler === 'number' ? otherHandler : otherHandler.handlerTag);
}
```

Also I think the `void` can be removed in that case.

Reply:
I like them, it looks cleaner. https://github.com/software-mansion/react-native-gesture-handler/pull/3713/commits/98ce5c9be6c98294b217bd62c2d45cfa1dc3466a